sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -3809,13 +3809,13 @@
           "calculateServer": true
         },
         {
-          "input": true,
-          "tableView": false,
-          "key": "dependantstatus",
           "label": "Dependant Status",
-          "type": "hidden",
+          "calculateValue": "const isStudentIndependant =\r\n  data.hasDependents === \"yes\" ||\r\n  [\"married\", \"other\", \"marriedUnable\"].includes(data.relationshipStatus) ||\r\n  data.outOfHighSchoolFor4Years === \"yes\" ||\r\n  data.fulltimelabourForce === \"yes\" ||\r\n  data.parentsDeceased === \"yes\" ||\r\n  [\"irreconcilableRift\", \"unableToContact\"].includes(data.parentSituation)\r\n  data.custodyOfChildWelfare === \"yes\";\r\n// If the student is not independant, dependant status is calculated based on personal information answer inputs.\r\nif (!isStudentIndependant) {\r\n  const isPersonalInfoAnswered =\r\n    !!data.hasDependents &&\r\n    !!data.relationshipStatus &&\r\n    !!data.outOfHighSchoolFor4Years &&\r\n    !!data.fulltimelabourForce &&\r\n    !!data.parentsDeceased &&\r\n    !!data.estranged &&\r\n    !!data.youthInCare &&\r\n    (data.youthInCare === \"no\" || !!data.custodyOfChildWelfare) &&\r\n    (data.estranged === \"no\" || !!data.parentSituation);\r\n  // If the answers to required personal information inputs are not provided, set the dependant status to null.\r\n  // Otherwise, dependant status is calculated based on personal information answers.\r\n  const calculatedStatusFromPersonalInfo = isPersonalInfoAnswered\r\n    ? \"dependant\"\r\n    : null;\r\n  instance.setValue(calculatedStatusFromPersonalInfo);\r\n} else {\r\n  instance.setValue(\"independant\");\r\n}",
```

Comment:
Please take care of the missing `||` condition here. 
![image](https://github.com/user-attachments/assets/0262d4c2-0e2d-42e9-9829-d2d0d96720c0)

Reply:
Fixed.